scripts/stablex/cancel_order.js

```diff
@@ -8,7 +8,7 @@ const argv = require("yargs")
     describe: "Order IDs to be canceled",
     coerce: array => {
       try {
-        return array.flatMap(v => v.split(",").map(o => parseInt(o)))
+        return array[0].split(",").map(o => parseInt(o))
```

Comment:
This only works if the array always only have 1 item. If this is the case it should probably not be an array in the first place but instead a string?

Reply:
That makes sense. Although why would the array every have more than one Item... I will fix it nonetheless